docs/content/BorderBox.md

```diff
@@ -1,16 +1,19 @@
-# BorderBox
+---
+title: BorderBox
+---
+
 
 BorderBox is a Box component with a border. When no `borderColor` is present, the component defaults to a gray border.
 
 ## Default example
 
-```.jsx
+```jsx live live
```

Comment:
Do these need to be "live live"? Maybe a find/replace typo?
Doesn't look like it's hurting anything though.